Makefile

```diff
@@ -0,0 +1,32 @@
+JSONNET_FMT := jsonnet fmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+
+all: fmt prometheus_alerts.yaml prometheus_rules.yaml dashboards_out lint test
```

Comment:
I don't think we will have these kind of targets. This will be a pure jsonnet library. Maybe as examples we can generate YAML.